packages/icons-react-native/scripts/index.js

```
@@ -30,17 +30,32 @@ fonts.forEach((name) => {
   };
   new WebpackIconfontPluginNodejs(options).build(() => {
-    const json = require(path.join("../", options.jsOutput));
+    // Read the generated JavaScript file and execute it to get the data
+    const jsContent = fs.readFileSync(options.jsOutput, 'utf8');
+    // Convert ES6 export to CommonJS and execute
+    const commonJsContent = jsContent
+      .replace(/\/\*[\s\S]*?\*\//g, '') // Remove comments
+      .replace(/export\s+default\s+/, 'module.exports = ');
+    
+    // Execute the modified JavaScript
+    const json = eval(commonJsContent);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)